fuse_models/include/fuse_models/unicycle_3d_state_cost_function.h

```diff
@@ -255,7 +256,6 @@ class Unicycle3DStateCostFunction : public ceres::SizedCostFunction<15, 3, 4, 3,
         // Also see: https://github.com/ceres-solver/ceres-solver/issues/387
 
         Eigen::Map<fuse_core::Matrix<double, 15, 4>> jacobian(jacobians[1]);
```

Comment:
Well if jacobians is 15x4 -> it should have already the size it needs so the index would need to be 0

Reply:
I am sorry but I do not really understand your comment. 
Please note that `jacobians` is an array with arrays inside and `jacobians[1]` is the jacobian of the second element (the orientation) which is already 15x4 large array allocated by ceres itself.

Reply:
Also not the case but we discussed it now in person.
To have the misunderstanding possibly also resolved for future readers, I would like to highlight more the structure of the function parameter `jacobians`:
`jacobians` contains the jacobian matrices for all of the fuse/ceres variables meaning that for example in this case:
- `jacobians[0]` contains the full jacobian matrix (represented as flat array) for the position1 (3 dimensions with overall 15 residuals so `jacobians[0]` is a flat simple array with 45 entries which can be remapped using Eigen as 15x3 matrix)
- `jacobians[1]` contains the full jacobian matrix (represented as flat array) for the orientation1 (4 dimensions with overall 15 residuals so `jacobians[1]` is a flat simple array with 60 entries which can be remapped using Eigen as 15x4 matrix)
- `jacobians[2]` contains the full jacobian matrix (represented as flat array) for the linear_velocity1 (3 dimensions with overall 15 residuals so `jacobians[2]` is a flat simple array with 45 entries which can be remapped using Eigen as 15x3 matrix)
- etc.